Makefile

```diff
@@ -49,4 +52,4 @@ build-windows: imports
 
 internal/test/e2e/tailor-test: cmd/tailor/main.go go.mod go.sum pkg/cli/* pkg/commands/* pkg/openshift/* pkg/utils/*
 	@(echo "Generating E2E test binary ...")
-	@(cd cmd/tailor && go build -o ../../internal/test/e2e/tailor-test)
+	@(cd cmd/tailor && go build -gcflags "all=-trimpath=$(CURDIR);$(shell go env GOPATH)" -o ../../internal/test/e2e/tailor-test)
```

Comment:
What do you need this for?

Reply:
It's not required, but I wanted to align it with the standard build, see https://github.com/opendevstack/tailor/pull/196/commits/9c6d0db9baf672f97ff1521301601a81435a967b.

That removes paths from my filesystem from the binary, otherwise you see e.g. `/Users/michaelsauter/...` when there is a stack trace ...